models/stg_stripe.yml

```diff
@@ -14,7 +14,7 @@ models:
         tests:
           - not_null
       - name: amount
-        description: Gross amount of the transaction, in cents.
+        description: Gross amount of the transaction, in currency units.
```

Comment:
What does currency units refer to in this context? For some reason this makes me think whole number which I know is not correct.

Reply:
I updated the documentation so that this is addressed by `{{ doc('convert_values') }}` referencing the docs.md. Lmk your thoughts about the language.